From c5d98cb0c10a231730ed3f895123c3aff9b79601 Mon Sep 17 00:00:00 2001
From: Gerrit Pape <pape@smarden.org>
Date: Wed, 8 Jan 2014 12:26:59 +0000
Subject: [JOBS] address format-security build error

Apply patch from Simon Ruderich <simon@ruderich.org> included in NMU from
Michael Gilbert.
---
 src/jobs.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/jobs.c b/src/jobs.c
index bf40204..f3af44d 100644
--- a/src/jobs.c
+++ b/src/jobs.c
@@ -427,7 +427,7 @@ sprint_status(char *s, int status, int sigonly)
 				goto out;
 #endif
 		}
-		col = fmtstr(s, 32, strsignal(st));
+		col = fmtstr(s, 32, "%s", strsignal(st));
 #ifdef WCOREDUMP
 		if (WCOREDUMP(status)) {
 			col += fmtstr(s + col, 16, " (core dumped)");
-- 
1.7.10.4

